helm/values.yaml

```diff
@@ -49,6 +44,10 @@ cron:
   concurrencyPolicy: Forbid
   restartPolicy: Never
 
+serviceUrls:
+  jobManager: 'http://job-service-url'
```

Comment:
shouldn't it be empty so it can be overrided from shared deployments?

Reply:
the shared-deployments umbrella overrides this value.